client/src/components/SmallContainer/SmallContainer.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import Container from '../Container/Container';
+
+const SmallContainer = (props) => {
+  return (
```

Comment:
You can skip curly brackets and return if there are only single return statement.